libcaf_openssl/src/manager.cpp

```diff
@@ -55,7 +58,19 @@ void manager::stop() {
 
 void manager::init(actor_system_config&) {
   CAF_LOG_TRACE("");
-  // TODO: initialize OpenSSL state
+
```

Comment:
[Vertical whitespaces separate functions and are not used inside functions: use comments to document logical blocks.](https://github.com/actor-framework/actor-framework/blob/master/CONTRIBUTING.md#general)